web/thesauruses/r/4/lists.json

```diff
@@ -0,0 +1,511 @@
+        "not-implemented":true
+      },
+      "insert_into_beginning_of_sorted_mutable_list": {
+        "name": "Insert element at beginning",
+        "not-implemented":true
+      },
+      "insert_into_end_of_sorted_mutable_list": {
+        "name": "Insert element at end",
+        "not-implemented":true
+      },
+      "insert_into_middle_of_sorted_mutable_list": {
+        "name": "Insert element in middle",
+        "not-implemented":true
+      },
+      "erase_element_at_beginning_of_sorted_mutable_list": {
+        "name": "Erase first element",
+        "not-implemented":true
+      },
+      "erase_element_at_end_of_sorted_mutable_list": {
+        "name": "Erase last element",
+        "not-implemented":true
+      },
+      "erase_element_in_middle_of_sorted_mutable_list": {
+        "name": "Erase element in the middle",
+        "not-implemented":true
+      },
+      "swap_elements_in_sorted_mutable_list": {
+        "name": "Swap two elements",
+        "not-implemented":true
+      },
+      "delete_sorted_mutable_list": {
+        "name": "Delete the list",
+        "not-implemented":true
+      },
+      "name_of_unordered_mutable_list": {
+        "name": "What is a unordered/unsorted mutable list called?",
+        "code": [
+          "list"
+        ]
+      },
+      "create_a_unordered_mutable_list": {
+        "name": "Create the list",
+        "code": [
+            "list()",
+            "mylist <- list(bob=1,sally=2)"
+          ]
+      },
+      "unordered_mutable_list_start_number": {
+        "name": "What number does it start at?",
+        "code": [
+          "1"
+        ]
+      },
+      "unordered_mutable_list_can_be_appended": {
+        "name": "Can you append to it?",
+        "code": [
+          "mylist <- list(a = 1, b = 2, c = 3)",
+          "mylist <- append(mylist, list(d = 4))"
+        ]
+      },
+      "unordered_mutable_list_can_be_inserted_in_middle": {
+        "name": "Can you insert into the middle of it?",
+        "code": [
+         "yes"
+        ]
+      },
+      "access_element_in_unordered_mutable_list": {
+        "name": "Access element by index",
+        "code": [
+          "mylist[[1]]"
+        ]
+      },
+      "insert_into_beginning_of_unordered_mutable_list": {
+        "name": "Insert element at beginning",
+        "code": [
+            "mylist <- list(a = 1, b = 2, c = 3)",
+            "mylist <- append(mylist, list(x = 42), 1)"
+        ]
+      },
+      "insert_into_end_of_unordered_mutable_list": {
+        "name": "Insert element at end",
+        "code": [
+            "mylist <- list(a = 1, b = 2, c = 3)",
+            "mylist <- append(mylist, list(x = 42), length(mylist))"
+        ]
+      },
+      "insert_into_middle_of_unordered_mutable_list": {
+        "name": "Insert element in middle",
+        "code": [
+            "mylist <- list(a = 1, b = 2, c = 3)",
+            "mylist <- append(mylist, list(x = 42), 2)"
```

Comment:
Is it safe to assume the `,2` is to insert in position 2? But also does this start at 0 or at 1, or is it based on the numbering from the list on the line above? Maybe add a `comment:` to clarify?